pkg/controller/container-runtime-config/helpers.go

```diff
@@ -954,3 +1007,65 @@ func generateSigstoreRegistriesdConfig(clusterScopePolicies map[string]signature
 	}
 	return data, nil
 }
+
+func addScopeMirrorsSigstoreRegistriesdConfig(registriesDockerConfig map[string]dockerConfig, scope string, reg *sysregistriesv2.Registry, sigstoreAttachment dockerConfig) error {
+	if reg == nil {
+		return nil
+	}
+	dummyDigest := "sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+	dummyTag := "aaa"
+	dummyPath := "/a"
+	dummyPrefix := "matched"
+	addedDummy := false
+	hasWildcard := false
+	originalScope := scope
+
+	if strings.HasPrefix(scope, "*.") {
+		scope = strings.Replace(scope, "*", dummyPrefix, 1)
+		hasWildcard = true
+	}
+
+	if !strings.Contains(scope, "/") {
+		scope += dummyPath
+		addedDummy = true
+	}
+
+	scopeRef, err := reference.Parse(scope)
+	if err != nil {
+		return fmt.Errorf("error parsing scope %s: %w", scope, err)
+	}
+
+	repo := reference.TrimNamed(scopeRef.(reference.Named))
```

Comment:
Non-blocking: This cast _probably_ can’t fail, but double-checking might be easier than worrying about crashes.